tests/integration/conftest.py

```diff
@@ -109,7 +120,7 @@ def charm_versions() -> IntegrationTestsCharms:
                 "revision": 70,
                 "series": "jammy",
                 "alias": "zookeeper",
-                "num_units": 3,
+                "num_units": 1,
```

Comment:
+1 on reducing the number of units. I also think we should rethink a little bit on the tests related to zookeeper in the near future now that we have changed the role of zookeeper in our bundle (it exists for service discovery and not for HA, but our current tests revolve around Zookeeper HA)